db/seeds.rb

```diff
@@ -1,20 +1,39 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
 
 Category.create(name:'Comedies')
 Category.create(name:'Dramas')
 Category.create(name:'Action')
 Category.create(name:'Sci-Fi')
 
-Video.create name:'Monk', description:'asdasdsda', cover_small_url:'monk_small.jpg', category_id:1
+arr_videos = []
 
-Video.create name:'Family Guy', description:'asdasdsda', cover_small_url:'family_guy.jpg',category_id:2
+# Generate 25 videos with random cover art and placed in categories at random.
+25.times do
+  category_num = (Random::rand * 4).ceil
+  cover_num = ((Random::rand * 4).ceil)
 
-Video.create name:'Futurama', description:'asdasdsda',  cover_small_url:'futurama.jpg', category_id:3
+  case cover_num
+  when 1
+    cover = 'monk_small.jpg'
+  when 2
+    cover = 'family_guy.jpg'
+  when 3
+    cover = 'futurama.jpg'
+  when 4
+    cover = 'south_park.jpg'
+  end
+  arr_videos << Fabricate(:video, name: Faker::Lorem.sentence, cover_small_url: cover, category_id: category_num)
 
-Video.create name:'South Park', description:'asdasdsda', cover_small_url:'south_park.jpg', category_id:1
+end
+
+arr_users = []
+# Generate 5 unique users.
+5.times do
+  arr_users << Fabricate(:user)
```

Comment:
You can use `Fabricate.times(5, :user)` to create multiple x times users. Also you don't need to `<<` to the array as it will return an array of those users.